server/src/main/java/com/paypal/selion/utils/SauceLabsRestApi.java

```diff
@@ -142,14 +204,49 @@ public int getMaxConcurrency() {
         LOGGER.entering();
         if (maxTestCase == -1) {
             try {
-                String result = getSauceLabsRestApi(sauceUrl + "/limits");
-                JsonObject obj = new JsonParser().parse(result).getAsJsonObject();
+                SauceLabsHttpResponse result = doSauceRequest("/limits");
+                JsonObject obj = result.getEntityAsJsonObject();
                 maxTestCase = obj.get("concurrency").getAsInt();
-            } catch (JsonSyntaxException | IllegalStateException e) {
-                LOGGER.log(Level.SEVERE, e.getMessage(), e);
+            } catch (JsonSyntaxException | IllegalStateException | IOException e) {
+                LOGGER.log(Level.SEVERE, "Unable to get max concurrency.", e);
             }
         }
         LOGGER.exiting(maxTestCase);
         return maxTestCase;
     }
+
+    private void addToAccountCache(String authKey, boolean valid) {
+        if (accountCache.size() >= 10) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2418638937/issues/source?bid=2928425&fileBranchId=3233060#l219)